### ISSUE TITLE
Usage of the virtual machine service account for cloud operations

### ISSUE THREAD

**Issue by adushein** [yandex-cloud/docker-machine-driver-yandex#18]
We need the ability to use a service account bound to master VM to create virtual docker-machines in the cloud.
I tried using a wrapper for docker-machine
```
#!/bin/bash
yc_token=$(curl -s -H 'Metadata-Flavor: Google' 'http://169.254.169.254/computeMetadata/v1/instance/service-accounts/default/token' | sed -r -e 's/.*"access_token": ?"([^"]+)".*/\1/')
set -- $(echo $@ | sed -r -e "s/__YC_TOKEN__/$yc_token/")
exec $(dirname $0)/true-docker-machine $@
```

but it doesn't work for deletion of the docker-machine VM after 12 hours, since iam token is saved in the docker-machine metadata and it isn't updated.

**Comment by GennadySpb** [comment]
Looks like the driver needs to support instance-based authentication, is it right?

**Comment by valyankilyan** [comment]
Implemented in [v0.1.36](https://github.com/yandex-cloud/docker-machine-driver-yandex/releases/tag/v0.1.36)
